markdown/docs/addon-author-guide.md

```diff
@@ -37,19 +38,21 @@ Node, this value is not set.
 
 ### Tracking It Down
 
-FastBoot works by turning an Ember application and all of its
-dependencies, including your addon, into a single bundle of JavaScript.
-Similar to how you ship an `app.js` and `vendor.js` file to the browser
-when you deploy an Ember app, the FastBoot server also loads these two
-files.
+FastBoot works by running a singular build as it was running without the FastBoot
+addon being installed. It basically loads the same assets `app.js` and `vendor.js`
```

Comment:
Perhaps this could be simplified a bit.

```FastBoot works by running a singular build of your application, leveraging the common `app.js` and `vendor.js` files on the server and browser clients. However, on the server an additional asset `app-fastboot.js` will leveraged. This asset contains FastBoot specific overrides defined by the application or other addons.```